Assets/Game/Application.cs

```diff
@@ -30,11 +30,11 @@ protected override void LoadModules()
             Container.LoadModule<ComputedModule>();
         }
 
-        protected override void StartSystems()
-        {
-            this.BindAllSystemsWithinApplicationScope();
-            this.StartAllBoundSystems();
-        }
+        //protected override void StartSystems()
```

Comment:
Can we get the comments removed, otherwise they will just end up hanging around.